domain/schema/controller.go

```diff
@@ -86,6 +90,8 @@ func ControllerDDL() *schema.Schema {
 		triggers.ChangeLogTriggersForSecretBackendRotation("backend_uuid", tableSecretBackendRotation),
 		triggers.ChangeLogTriggersForModelSecretBackend("model_uuid", tableModelSecretBackend),
 		triggers.ChangeLogTriggersForModel("uuid", tableModelMetadata),
+		triggers.ChangeLogTriggersForModelAuthorizedKeys("model_uuid", tableModelAuthorizedKeys),
```

Comment:
Happy to do this but it would be a big change from what we have today. Today we just emit notify changes to the agent and it reloads the world. Not idea but it is  what it is.

If we swap to watching changes based on keys the advantage of this is that we can start watching on key changes and provide a strings watcher and the value to be removed back down to the agent. This would require a lot of code change in the agent.

More specifically it would still not allow us to deal with deleted keys with the trigger because we don't write our own trigger and can't control the value into the change log.

Happy to do this just can't see how it will help us today unless we are saying we want 2 very big changes to the agent work for authorized keys and also implement custom triggers for the change log. Without those two things we are potentially just opening up the fire hose on the watcher  for change events that it doesn't want to know about.